src/main/java/picard/sam/RevertSam.java

```diff
@@ -269,7 +269,7 @@ protected int doWork() {
         }
 
         final boolean sanitizing = SANITIZE;
-        final SamReader in = SamReaderFactory.makeDefault().referenceSequence(referenceSequence.getReferencePath()).validationStringency(VALIDATION_STRINGENCY).open(SamInputResource.of(INPUT.toPath())); // tsato: confirm this won't break piped input
+        final SamReader in = SamReaderFactory.makeDefault().referenceSequence(referenceSequence.getReferencePath()).validationStringency(VALIDATION_STRINGENCY).open(SamInputResource.of(INPUT.toPath()));
```

Comment:
@takutosato Could you put a comment here referencing this PR or the related issue?  Just too save us some time when we forget and someone tries to change it.

Reply:
Done

Reply:
👍